functions.inc.php

```diff
@@ -210,7 +210,7 @@ function callrecording_get_config($engine) {
 		$ext->add($context, $exten, 'stoprec', new ext_noop('Stopping recording: ${ARG2}, ${ARG3}'));
 		$ext->add($context, $exten, '', new ext_set('__REC_STATUS','STOPPED'));
 		// See https://issues.asterisk.org/jira/browse/ASTERISK-24527
-		$ext->add($context, $exten, '', new ext_system(FreePBX::Config()->get('ASTVARLIBDIR').'/bin/stoprecording.php "${CHANNEL(name)}"'));
+		$ext->add($context, $exten, '', new ext_system('${AMPBIN}/stoprecording.php "${CHANNEL(name)}"'));
```

Comment:
AMPBIN is defined as a global in extensions_additional.conf, which is
included at the top of extensions.conf.

That was done in this pull: https://github.com/FreePBX/core/pull/14